scripts/zones/North_Gustaberg_[S]/npcs/Lycopodium.lua

```diff
@@ -0,0 +1,29 @@
+-----------------------------------
+-- Area: North Gustaberg [S]
+--  NPC: Lycopodium
+-- !pos -275.953,12.333,262.368 88
```

Comment:
Consistency demands spaces and no commas...!

Reply:
fixed! don't shoot the messenger, shoot the commas lol 🐱